src/shared/components/query/CancerStudySelector.tsx

```diff
@@ -8,9 +8,9 @@ import classNames from 'classnames';
 import ReactSelect from 'react-select';

```

Comment:
had to reformat this file heavily b/c andy was using very different formatting settings.  so diff includes a lot of code which is reformatted and not refactored

Reply:
could you point out which lines are new code?

Reply:
nevermind